AIO/src/org/aio/activities/quests/RomeoAndJuliet.java

```diff
@@ -53,11 +56,6 @@ public void runActivity() throws InterruptedException {
             return;
         }
 
-        if (getTabs().getOpen() != Tab.INVENTORY) {
-            getTabs().open(Tab.INVENTORY);
-            return;
-        }
```

Comment:
This part produces the deadlock, i wasnt sure how to link it

Reply:
Atleast with `-allow norandoms` i dont use the randomsolvers

Reply:
You said you got stuck on progress 50, if this caused you to get stuck you would have been stuck from the beginning. The reason it got stuck on 50 was because of an if statement that checked if the quest tab was open. #34

Reply:
Yeah because this is the first cutscene -> my config was 50 when i checked but with log statements i found out i got stuck there because of the cutscene.

For me it feels like im touching your baby lol

Reply:
Haha, no no. Im just trying to understand the changes. Since I was trying to get the cut scenes to work simply. So it's great if this just works 👍